messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/logging/MessagingLogger.java

```diff
@@ -819,4 +819,6 @@
     @Message(id = 87, value = "Unable to load connector service factory class: %s")
     OperationFailedException unableToLoadConnectorServiceFactoryClass(String factroyClass);
 
+    @Message(id = 88, value = "Invalid journal-file-size %s , It should be multiple of 512")
```

Comment:
The attribute name should not be hard-coded. THis exception is thrown by  ModularLongRangeParameterValidator which could be used by other attributes than journal-file-size